packages/kit/src/runtime/server/page/load_node.js

```diff
@@ -3,9 +3,10 @@ import * as set_cookie_parser from 'set-cookie-parser';
 import { normalize } from '../../load.js';
 import { respond } from '../index.js';
 import { LoadURL, PrerenderingURL, is_root_relative, resolve } from '../../../utils/url.js';
-import { check_method_names, is_pojo, lowercase_keys } from '../utils.js';
+import { check_method_names, lowercase_keys } from '../utils.js';
 import { coalesce_to_error } from '../../../utils/error.js';
 import { domain_matches, path_matches } from './cookie.js';
+import { is_pojo, warn_if_not_serdeable } from './utils.js';
```

Comment:
TIL that serdeable is an actual word 😄

Reply:
Serde is common in the rust circles. Guess it hasn't carried over much outside, will change if it's not clear.